### ISSUE TITLE
Plugin crash with java 11

### ISSUE THREAD

**Issue by asbachb** [warmuuh/libsass-maven-plugin#83]
I wonder if anyone is using the plugin with java 11? For the the plugin crashes on execution:

```
JAVA_HOME=C:\\dev\\bin\\java\\adoptopenjdk\\_hotspot\\jdk-11.0.8+10
...
--- libsass-maven-plugin:0.2.10-libsass_3.5.3:compile (default) @ project1 ---
[buildinfo] Properties file found at 'C:\Users\benjamin.asbach\AppData\Local\Temp\buildInfo1736748084316620382.properties'
```

**Comment by asbachb** [comment]
```
[DEBUG] Created new class realm plugin>com.github.warmuuh:libsass-maven-plugin:0.2.10-libsass_3.5.3
[DEBUG] Importing foreign packages into class realm plugin>com.github.warmuuh:libsass-maven-plugin:0.2.10-libsass_3.5.3
[DEBUG]   Imported:  < maven.api
[DEBUG] Populating class realm plugin>com.github.warmuuh:libsass-maven-plugin:0.2.10-libsass_3.5.3
[DEBUG]   Included: com.github.warmuuh:libsass-maven-plugin:jar:0.2.10-libsass_3.5.3
[DEBUG]   Included: io.bit3:jsass:jar:5.7.1
[DEBUG]   Included: org.apache.commons:commons-lang3:jar:3.7
[DEBUG]   Included: commons-io:commons-io:jar:2.6
[DEBUG]   Included: org.apache.commons:commons-text:jar:1.3
[DEBUG]   Included: javax.enterprise:cdi-api:jar:1.0
[DEBUG]   Included: com.google.guava:guava:jar:10.0.1
[DEBUG]   Included: com.google.code.findbugs:jsr305:jar:1.3.9
[DEBUG]   Included: org.sonatype.sisu:sisu-guice:jar:no_aop:3.1.0
[DEBUG]   Included: aopalliance:aopalliance:jar:1.0
[DEBUG]   Included: org.eclipse.sisu:org.eclipse.sisu.inject:jar:0.0.0.M5
[DEBUG]   Included: org.eclipse.aether:aether-util:jar:0.9.0.M2
[DEBUG]   Included: org.codehaus.plexus:plexus-interpolation:jar:1.19
[DEBUG]   Included: org.codehaus.plexus:plexus-utils:jar:3.0.15
[DEBUG]   Included: org.codehaus.plexus:plexus-component-annotations:jar:1.5.5
[DEBUG]   Included: org.sonatype.plexus:plexus-sec-dispatcher:jar:1.3
[DEBUG]   Included: org.sonatype.plexus:plexus-cipher:jar:1.4
[DEBUG]   Included: org.webjars:webjars-locator-core:jar:0.32
[DEBUG]   Included: org.apache.commons:commons-compress:jar:1.9
[DEBUG]   Included: com.fasterxml.jackson.core:jackson-core:jar:2.7.3
[DEBUG]   Included: org.sonatype.plexus:plexus-build-api:jar:0.0.7
[DEBUG] Configuring mojo com.github.warmuuh:libsass-maven-plugin:0.2.10-libsass_3.5.3:compile from plugin realm ClassRealm[plugin>com.github.warmuuh:libsass-maven-plugin:0.2.10-libsass_3.5.3, parent: jdk.internal.loader.ClassLoaders$AppClassLoader@946e09]
[DEBUG] Configuring mojo 'com.github.warmuuh:libsass-maven-plugin:0.2.10-libsass_3.5.3:compile' with basic configurator -->
[DEBUG]   (f) copySourceToOutput = false
[DEBUG]   (f) embedSourceContentsInSourceMap = false
[DEBUG]   (f) embedSourceMapInCSS = false
[DEBUG]   (f) enableClasspathAwareImporter = false
[DEBUG]   (f) failOnError = true
[DEBUG]   (f) generateSourceComments = false
[DEBUG]   (f) generateSourceMap = false
[DEBUG]   (f) inputPath = src/main/sass
[DEBUG]   (f) inputSyntax = scss
[DEBUG]   (f) omitSourceMapingURL = false
[DEBUG]   (f) outputPath = C:\dev\src\java\project1\target\project1-7.1.0-SNAPSHOT\css
[DEBUG]   (f) outputStyle = compressed
[DEBUG]   (f) precision = 5
[DEBUG]   (f) project = MavenProject: com.amadeus.rail:project1:7.1.0-SNAPSHOT @ C:\dev\src\java\project1\pom.xml
[DEBUG]   (f) sourceMapOutputPath = C:\dev\src\java\project1\target
[DEBUG] -- end configuration --
[DEBUG] Input Path=src/main/sass
[DEBUG] Output Path=C:\dev\src\java\project1\target\project1-7.1.0-SNAPSHOT\css
[DEBUG] Glob = glob:{**/,}*.scss
[DEBUG] Processing File C:\dev\src\java\project1\src\main\sass\admin.scss
```

**Comment by asbachb** [comment]
This problem only occurs with 32bit jdk/os. Seesm to be a limitation by used jsass library: https://github.com/bit3/jsass#compatibility-overview
